inventory/migrations/0001_initial.py

```diff
@@ -32,10 +33,14 @@ class Migration(migrations.Migration):
             name='Ownership',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('added_since', models.DateField(default=datetime.date.today)),
+                ('added_since', models.DateField(default=django.utils.timezone.now)),
```

Comment:
I don't think we should modify migrations that are already live in production?

Reply:
Agreed, except that this file shouldn't exist at all as it was used in the squashedmigration. But it was incorrectly left here and as a result I couldn't make a single change to Ownership as the intermediate files have been removed (this problem caught my attention last monday when I spend nearly two hours figuring out why the model registry failed when trying to migrate). If you want to you can try to recreate it on a clean branch and see if you don't have any issues with it on your local database, but this was the solution I came up with to fix the migration registry